src/components/organisms/Room/Participant.scss

```
@@ -0,0 +1,59 @@
+@import "scss/constants";
+
+.participant {
+  background-color: $light-grey;
+  display: flex;
+  justify-content: center;
+  justify-self: center;
+  max-width: $video-participant-optimal-width;
+  width: $video-participant-optimal-width;
+  height: $video-participant-optimal-height;
+  max-height: $video-participant-optimal-height;
+  display: flex;
+  padding: 0;
+
+  video {
+    height: 100%;
+    width: 100%;
+  }
+
+  .mirrored {
+    transform: scale(-1, 1);
+  }
+
+  .profile-icon {
+    position: absolute;
+    bottom: $spacing--sm;
+    left: $spacing--sm;
+    border-radius: 188%;
+    z-index: z(jazzbar-participant-profile-icon);
+
+    &__onlyIcon {
```

```suggestion
    &--only-icon {
```

but not something that results in `&--only-icon--remote`

---

ended up having `-icon--remote` there c4879fa